src/Package/Impl/DataInspect/DataImport/ImportDataWindow.xaml.cs

```diff
@@ -164,35 +164,23 @@ public ImportDataWindow(string filePath, string name)
             }
         }
 
-        private async Task RunAsync(string expression) {
+        private bool Execute(string expression) {
             try {
-                REvaluationResult result = await EvaluateAsync(expression);
-                if (result.ParseStatus == RParseStatus.OK && result.Error == null) {
-                    Close();
-                } else {
-                    OnError(result.ToString());
-                }
+                var workflow = VsAppShell.Current.ExportProvider.GetExportedValue<IRInteractiveWorkflowProvider>().GetOrCreate();
+                workflow.Operations.ExecuteExpression(expression);
+                return true;
```

Comment:
Ok

---
In reply to: [60455182](https://github.com/Microsoft/RTVS/pull/1651#discussion_r60455182) [](ancestors = 60455182)